amr-wind/utilities/console_io.cpp

```diff
@@ -92,9 +92,9 @@ void print_banner(MPI_Comm comm, std::ostream& out)
 
     auto etime = std::chrono::system_clock::now();
     auto etimet = std::chrono::system_clock::to_time_t(etime);
-    char time_buf[64];
-    ctime_r(&etimet, time_buf);
-    const std::string tstamp(time_buf);
+    amrex::Array<char, 64> time_buf;
```

Comment:
Is an amrex array the right format for this variable?

Reply:
basically we were using raw c-style arrays. Now we use an amrex class that is a c++ array and implements all the necessary checks so we don't go out of bounds, etc. It's safer this way.